pkg/k8sclient/k8sclient.go

```diff
@@ -24,7 +24,9 @@ import (
 	"strings"
```

Comment:
it possibly need some declaration  for changing the original code,  owing to license issue